Dockerfile

```
@@ -58,10 +61,11 @@ RUN gradle --console=plain \
 
 FROM openjdk:11.0.6-jre
 ARG hub_app
+ARG conf_dir
```

```suggestion
ARG conf_dir=configuration
```
If we delete the lines at the top of the file, we need to reintroduce the default value here.